StationeersLaunchPad/LaunchPadPlugin.cs

```diff
@@ -294,18 +294,25 @@ static bool GameString_op_Implicit(ref string __result, GameString gameString)
       return false;
     }
 
+    private static string _savePathOverride;
     [HarmonyPatch(typeof(StationSaveUtils), nameof(StationSaveUtils.DefaultPath), MethodType.Getter), HarmonyPrefix]
     static bool StationSaveUtils_DefaultPath(ref string __result)
     {
-      if (LaunchPadConfig.SavePathOverride.Value == null)
+      _savePathOverride ??= LaunchPadConfig.SavePathOverride.Value != null
+        ? LaunchPadConfig.SavePathOverride.Value
+        : string.Empty;
+      
+      if (_savePathOverride == "")
```

Comment:
Btw, the reason for using an empty string instead of null for this value is so that null can represent the cached value not being set yet.  This is a cleaner method than for example using a string pointer (or string array with a single value).